pkgs/applications/misc/apvlv/default.nix

```diff
@@ -52,6 +52,11 @@ stdenv.mkDerivation rec {
     mkdir -p $out/share/doc/apvlv/
     cp ../Startup.pdf $out/share/doc/apvlv/Startup.pdf
     cp ../main_menubar.glade $out/share/doc/apvlv/main_menubar.glade
+  ''
+  + stdenv.lib.optionalString (!stdenv.isDarwin) ''
+    # copy and patch the original apvlv.desktop file
```

Comment:
You mention patching, but I don't see any (explicit) patching here?

Reply:
I pushed an update with a more elegant "install -D"